README.md

```diff
@@ -16,6 +16,99 @@ Run the following commands to perform a release:
     $ mvn release:prepare release:perform
     $ git push --all && git push --tags
 
+## Parameter Store
+
+Parameter naming
+
+The parameter names are made from the following:
+* Project Name
+* Environment (dev/prod/sandbox/etc)
+* Date that the value was added in YYYYMMDD format
+* Parameter key
+Example: /weapon-x/dev/20190130/subject
+
+Log into Jenkins and upload your AWS SSM Parameter Store json file to the Managed Files section of Manage Jenkins.
+
+The format should look something like this:

```

Comment:
Suggest using terraform or manual entry for things like this. Eventually kms may give another automated solution.